Makefile

```diff
@@ -0,0 +1,95 @@
+# This makefile bundles commands for streamlined development
+# Use `make help` for an overview of commands
+
+conan_profile_name=tket
+tket_remote_repository=https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs
+build_type=Debug
+ifeq ($(build_type), Release)
+  conan_build_dir=cmake-build-release
+else
+  conan_build_dir=cmake-build-debug
+endif
+
+##@ General
+
+
+.PHONY: help
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Setup Conan Profile
+
+.PHONY: conan-profile
+conan-profile: ## Create and configure conan profile
+	-conan profile new $(conan_profile_name) --detect
+	-conan remote add tket-libs $(tket_remote_repository)
+	conan config set general.revisions_enabled=1
+	conan profile update options.tket:shared=True $(conan_profile_name)
+	conan profile update options.tklog:shared=True $(conan_profile_name)
+	conan profile update settings.build_type=$(build_type) $(conan_profile_name)
+
+##@ Install Dependencies
+
+.PHONY: pybind
+pybind: ## install custom pybind package
+	conan profile update settings.build_type=$(build_type) $(conan_profile_name)
+	conan remove -f "pybind11/*"
+	conan create --profile=$(conan_profile_name) recipes/pybind11
+
+ # Currently the b2 package (a dependency of boost) in the remote tket-libs does not include compiler info
```

Comment:
This may be fixed now? I rebuilt it a couple of days ago with the latest recipe from the conan-center.

Reply:
Problem persists so I'll leave it for now